bin/plugin/commands/test/__snapshots__/changelog.js.snap

```diff
@@ -0,0 +1,222 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`formatChangelog verify that the changelog is properly formatted 1`] = `
+"### Enhancements
+
+#### Components
+- components: Use updated range styles. ([33824](https://github.com/WordPress/gutenberg/pull/33824))
+- Components: Update snackbar to use framer motion instead of react spring. ([33717](https://github.com/WordPress/gutenberg/pull/33717))
+- components: Add new ColorPicker. ([33714](https://github.com/WordPress/gutenberg/pull/33714))
+- Components: Promote \`ItemGroup\`. ([33701](https://github.com/WordPress/gutenberg/pull/33701))
+
+#### Block Library
+- [Post Featured Image]: Add basic dimension controls. ([31634](https://github.com/WordPress/gutenberg/pull/31634))
+
+#### Site Editor
+- Site Editor: Add error boundary. ([33921](https://github.com/WordPress/gutenberg/pull/33921))
+
+#### Widgets Editor
+- Add error boundaries to widget screens. ([33771](https://github.com/WordPress/gutenberg/pull/33771))
+
+#### Patterns
+- Block patterns: Add translation context on titles. ([33734](https://github.com/WordPress/gutenberg/pull/33734))
+
+#### Document Settings
+- Disable spellcheck and autocomplete in permalink slug field. ([33708](https://github.com/WordPress/gutenberg/pull/33708))
+
+#### Template Editor
+- Template Mode: Add busy state to template creation modal. ([33076](https://github.com/WordPress/gutenberg/pull/33076))
+
+#### Global Styles
+- Dimensions Panel: Add new ToolsPanel component and update spacing supports. ([32392](https://github.com/WordPress/gutenberg/pull/32392))
+
+#### Uncategorized
```

Comment:
This is out of date with `trunk` as of https://github.com/WordPress/gutenberg/pull/34037